### PR TITLE
[chore] update readme, fix examples, show effects button

### PR DESCRIPTION
**Pull request by igorDykhta**
- update readme
- fix examples
- show effects button

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ You can add the script tag to your html file as it follows:
 or if you would like, you can load a specific version
 
 ```html
-<script src="https://unpkg.com/kepler.gl@0.2.2/umd/keplergl.min.js" />
+<script src="https://unpkg.com/kepler.gl@2.5.5/umd/keplergl.min.js" />
```
Comment (chrisgervang) [inline README.md:76]: v3 or continue using 2?

File: README.md [hunk 2]
```diff
@@ -165,11 +160,10 @@ stored in `state.keplerGl.foo`.
 In case you create multiple kepler.gl instances using the same id, the kepler.gl state defined by the entry will be
 overridden by the latest instance and reset to a blank state.
 
-##### `mapboxApiAccessToken` (String, required*)
+##### `mapboxApiAccessToken` (String, required\*)
```
Comment (chrisgervang) [inline README.md:163]: Typo or intentional?

File: examples/umd-client/index.html [hunk 3]
```diff
@@ -32,9 +45,9 @@
        * Provide your MapBox Token
        **/
       const MAPBOX_TOKEN = 'PROVIDE_MAPBOX_TOKEN';
```
Comment (chrisgervang) [inline examples/umd-client/index.html:47]: Do we use maplibre now?

**Review by chrisgervang — APPROVED** [review]

File: examples/umd-client/index.html (hunk 3 above)
Comment (birkskyum) [reply examples/umd-client/index.html:47]: The mapbox token was used for different things

- **allow mapbox-gl library to do a renderer**. `maplibre-gl` doesn't need a token for this local operation, and can readily be used offline if need be with a set of downloaded tiles.
- **access mapbox styles from style api (and map tiles)**. MapLibre doesn't have any services. We use the free Carto styles by default which doesn't need a token. If a paid style is to be used, like Stadia maps or MapTiler, an apikey will be part of the url when it's added, so here also we don't need a similar api key.

**Pull request merged**